HesoyamHospital/WebApplication/Appointments/AppointmentController.cs

```diff
@@ -10,10 +11,13 @@ public class AppointmentController : ControllerBase
     {
         private readonly IAppointmentService _appointmentService;
         private readonly long defaultPatientId = 500;
+        private readonly AppointmentValidation _appointmentValidation;
+        private readonly int hoursToCancelBeforeAppointment = 48;
```

Comment:
Business logic related variables should not stay in controller.
It should be encapsulated within the class which handles the validation if appointment can be canceled.

Reply:
Explaination: Let's say we have a controller for web app and a controller for mobile app.
The developers who work on a web app can set the value of variable _hoursToCancelBeforeAppointment_ to 48 hours, while the developers who work on mobile app controller set it to 24 hours.
As you can assume, we won't have consistency within our app anymore.